libr/bin/format/elf/elf.c

```diff
@@ -16,7 +16,7 @@
 #define IFDBG if (DO_THE_DBG)
 #define IFINT if (0)
 
-#define MIPS_PLT_OFFSET 108
+#define MIPS_PLT_OFFSET 0x20
```

Comment:
why this change?

Reply:
This value is incorrect for the current implementation